lib/ci-stack.ts

```diff
@@ -122,7 +125,7 @@ export class CIStack extends Stack {
       adminUsers: props?.adminUsers,
       agentNodeSecurityGroup: securityGroups.agentNodeSG.securityGroupId,
       subnetId: vpc.publicSubnets[0].subnetId,
-    }, agentNodes, agentAssumeRoleContext.toString());
+    }, agentNodes, agentAssumeRoleContext.toString(), macAgentParameter.toString());
```

Comment:
Why covert it to string? Also wondering if we should keep this just a property rather than command line parameter?
You need to pass all command line params for all cdk commands like `ls` `destroy`,`synth` etc